examples/combined-hbs-jsx/views/Home.jsx

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs'
+import Handlebars from 'handlebars'
+import path from 'path'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+import Component from './components/Component'
+
+const fileContent = fs.readFileSync(path.join(__dirname, './partials/partial-with-jsx-component.hbs'), 'utf8')
+
+export default function Home({ pageTitle }) {
+  function createMarkup(markup) {
+    return { __html: markup }
+  }
+
+  const template = Handlebars.compile(fileContent)
```

Comment:
You can hoist this safely out of the function so that it's only run once

Reply:
Fixed in this [commit](https://github.com/Financial-Times/dotcom-page-kit/pull/714/commits/a20b94be42dba050025fb722d23f9863d07fbfe0).